database/migrations/2021_12_09_171935_create_orders_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateOrdersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('orders', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('customer_id')
+            ->constrained()
+            ->restrictOnUpdate()
+            ->cascadeOnDelete();
+            $table->foreignId('tour_id')
+            ->constrained()
+            ->restrictOnUpdate()
+            ->cascadeOnDelete();
```

Comment:
looks as here cascade should be on update and restrict on delete

Reply:
Fixed